utils/mountd/cache.c

```diff
@@ -1429,6 +1435,52 @@ int cache_process_req(fd_set *readfds)
 	return cnt;
 }
 
+/*
+ * Expire export cache entries in path which are subdirectories of
+ * exp->m_export.e_path. For instance, if exp->m_export.e_path is
+ *     /a/b
+ * and path is
+ *     /a/b/c/d/e
+ * this will expire the export cach entries for
+ *     /a/b/c
+ *     /a/b/c/d
+ *     /a/b/c/d/e
+ */
+int cache_expire_subdirs(nfs_export *exp, char *path)
```

Comment:
Will the logic below work if the path passed in is /a/b/c//d/e or /a/b/c/d/e/? I'm not sure if this can happen but wanted to check for those types of corner cases.

Reply:
Thanks for verifying.